libraries/AC_WPNav/AC_Circle.h

```diff
@@ -20,9 +20,9 @@ class AC_Circle
     AC_Circle(const AP_InertialNav& inav, const AP_AHRS_View& ahrs, AC_PosControl& pos_control);
 
     /// init - initialise circle controller setting center specifically
-    ///     set terrain_alt to true if center.z should be interpreted as an alt-above-terrain
+    ///     set terrain_alt to true if center.z should be interpreted as an alt-above-terrain. Rate should be +ve in deg/sec for cw turn
     ///     caller should set the position controller's x,y and z speeds and accelerations before calling this
-    void init(const Vector3p& center, bool terrain_alt);
+    void init(const Vector3p& center, bool terrain_alt, float rate_deg_per_sec);
```

Comment:
I probably would have made the new init argument the direction rather than the rate but I guess it's ok.